analyzer/tools/build-logger/src/ldlogger-tool-gcc.c

```diff
@@ -364,16 +364,11 @@ int loggerGccParserCollectActions(
   char* keepLinkVar = getenv("CC_LOGGER_KEEP_LINK");
   int keepLink = keepLinkVar && strcmp(keepLinkVar, "true") == 0;
 
-  /* If prog_ is a relative path we try to
-   * convert it to absolute path.
-   */
-  path_ptr = realpath(prog_, full_prog_path);
-
-  /* If we cannot convert it, we try to find the
+  /* If toolName_ is not an absolute path, we try to find it as an
```

Comment:
Please add a comment here that we do not resolve symlinks on purpose, to support ccache use-case.